shpc/main/registry/remote.py

```diff
@@ -216,6 +214,21 @@ def _update_clone_cache(self):
             }
         shutil.rmtree(tmpdir)
 
+    def get_module_config_url(self, module_name):
+        """
+        Get the raw address of the config (container.yaml)
+        """
+        parts = self.url.split("/")
+        domain = parts[2]
+        if domain in self.raw_container_url_schemes:
+            t = ("/".join(parts[3:]), self.tag, module_name)
+            return (
```

Comment:
if you have a return like this for an if, you don't need the else - it will hit the else condition by not returning (just a design preference!)

Reply:
Changed